### DISCUSSION TITLE
FOAMySees

### DISCUSSION THREAD

**Discussion by yisangriB** [NHERI-SimCenter/SimCenterCommon#245] [Hydro-UQ]
Is FOAMySees available for free download, and where can it be downloaded for use with preCICE?

> (_The question is originally posted by the user @benjaminb115 and was deleted by accident, so we are recreating the post based on the notification email. Apologies to @benjaminb115_)

**Comment by fmckenna** [comment]
I am reaching out to the developer of FOAMySEES to see if she will give you access to her github repo or make it public.

**Reply by JustinBonus** [reply]
Try:
https://github.com/nsaoirse/FOAMySees/

Developer is Nikki Lewis, at Schnabel currently.

**Reply by JustinBonus** [reply]
If you don't receive access I can ask for permission to distribute.

**Comment by benjaminb115** [comment]
Thank you for your support. However, I encountered an error 404 when I
attempted to access https://github.com/nsaoirse/FOAMySees/.

**Comment by fmckenna** [comment]
Nikki has just kindly made the repo public. Try again. Thanks.
